src/integrations/admin/integrations-page.php

```diff
@@ -114,14 +113,16 @@ public function enqueue_assets() {
 
 		$this->admin_asset_manager->enqueue_script( 'integrations-page' );
 
-		$elementor_conditional = new Elementor_Activated_Conditional();
-		$jetpack_conditional   = new Jetpack_Conditional();
+		$elementor_conditional       = new Elementor_Activated_Conditional();
+		$jetpack_conditional         = new Jetpack_Conditional();
+		$google_site_kit_conditional = new Google_Site_Kit_Feature_Conditional();
```

Comment:
This should actually go via dependency injection. And now we connected the dots for the tests and the other two 😅